### ISSUE TITLE
Exception on initial cli.py script run

### ISSUE THREAD

**Issue by Pavel-Durov** [hrik2001/macuninstaller#3]
I tried to follow the README instructions but I got an exception on initial script run.

My exact steps with **Python 2.7.10**:

```
$ git clone https://github.com/hrik2001/macuninstaller.git
Cloning into 'macuninstaller'...
remote: Counting objects: 638, done.
remote: Compressing objects: 100% (317/317), done.
remote: Total 638 (delta 175), reused 463 (delta 154), pack-reused 154
Receiving objects: 100% (638/638), 8.08 MiB | 3.18 MiB/s, done.
Resolving deltas: 100% (232/232), done.

$ cd macuninstaller/

$ python cli.py /Applications/PlayOnMac.app
Traceback (most recent call last):
  File "cli.py", line 1, in <module>
    from function import *
  File ".../macuninstaller/function.py", line 5, in <module>
    import queue
ImportError: No module named queue

```
I tried to run the same script with **Python 3.6.3** and I go the following exception:

```
$ python3 cli.py /Applications/PlayOnMac.app
  File "cli.py", line 14
    print doc
            ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(doc)?

```

**Comment by hrik2001** [comment]
U forgot to put the p flag
`python cli.py -p /Applications/PlayOnMac.app`

Oh and macuninstaller doesn't have python 3 support

**Comment by hrik2001** [comment]
Oh and also looks like you have got the queue module missing, will u do me a favor and tell me what happens if you do this?

Open python interpreter and then type this
`import Queue`

If nothing happens then, in `function.py` edit the line `import queue` to `import Queue as queue`.

I will make a try except block In the next commit to help you out.

Make sure to put the p flag and keep me updated about the issue. :D

**Comment by Pavel-Durov** [comment]
Just ran with -p flag, and got these exceptions: 

**Python 2.7.10 :** 
```
$ python cli.py -p /Applications/PlayOnMac.app
Traceback (most recent call last):
  File "cli.py", line 1, in <module>
    from function import *
  File ".../macuninstaller/function.py", line 5, in <module>
    import queue
ImportError: No module named queue
```
 **Python 3.6.3 :**
```
$ python3 cli.py -p /Applications/PlayOnMac.app
  File "cli.py", line 14
    print doc
            ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(doc)?


```

**Comment by Pavel-Durov** [comment]
Regarding Queue module, I'm able to import it in **Python 2.7.10** but not in **Python 3.6.3**.
**Python 3.6.3 :** 
> The Queue module has been renamed to queue in Python 3. 
link: https://docs.python.org/2/library/queue.html

```
$ python3
Python 3.6.3 (v3.6.3:2c5fed86e0, Oct  3 2017, 00:32:08) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import queue as queue
>>> queue
<module 'queue' from '/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/queue.py'>
```

**Python 2.7.10 :** 
```
$ python
Python 2.7.10 (default, Jul 15 2017, 17:16:57) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.31)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import Queue as queue
>>> queue
<module 'Queue' from '/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/Queue.pyc'>

```

**Comment by Pavel-Durov** [comment]
**I have several inputs, let me know what you think:** 

* In case user misuses library interface (forgets adding -p flag as I did) it should output comprehensive error description.

* Obviously, there's a different behavior between python 2 and 3 - it should be handled in the same way or a decision should be made on which python version this project supports. Currently, there's no mention of python version in README, that's very confusing for the end user (I tried to run it first with python3, cause that's my default on my machine)

**Comment by hrik2001** [comment]
I saw your pull request but macs that dont have python3 dont have the `Queue` module. To be honest, `Queue` is a python3 module and not python2 and due to some reason, I think your python2 wasnt able to access `queue`. That is why I have added a try except block to import the queue module in whichever way it works the best. 
```
try:
        import queue
except ImportError:
        import Queue as queue
```
Also, macuninstaller does produce error messages if necessary flags are not put in the arguments. It is just that they were defined after `from function import *` , that is why you never saw them, here is a sample error message -

```
[macuninstaller]: ;_; ▶ python cli.py /Applications/PlayOnMac.app
usage: cli.py [-h] -p PATH [-c [CUSTOM [CUSTOM ...]]]
cli.py: error: argument -p/--path is required
[macuninstaller]: ;_; ▶ 
```
You are right about the python 3 and python 2 part. I am looking into it and will release the updated version soon :) . I ported it to python3 but the scan function was acting all weird. 

Lemme know if macuninstaller worked properly or not :) and thank you for being so supportive and taking time to write.
